### DISCUSSION TITLE
Intel arc770

### DISCUSSION THREAD

**Discussion by fablevi** [bghira/SimpleTuner#1198] [General]
Hello!

Is SimpleTuner works with intel arc card (a750, a770)? If yes, then how can i install it the right way? If not, then can you guys add it to the supported list? I want to try out your app:)

Thanks for help❤️

**Comment by bghira** [comment]
i don't have one but would be glad to support it. you will have to look at how `mps` is supported and probably follow similar style of considerations.

**Reply by fablevi** [reply]
Can you discribe what is mps?
